AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -82,7 +82,20 @@ - (void)_calculateSize
   CGRect constrainedRect = {CGPointZero, _constrainedSize};
   __block CGRect boundingRect;
   [_context performBlockWithLockedTextKitComponents:^(NSLayoutManager *layoutManager, NSTextStorage *textStorage, NSTextContainer *textContainer) {
+    NSParagraphStyle *paragraphStyle = [textStorage attributesAtIndex:[layoutManager characterIndexForGlyphAtIndex:0]
+                                                       effectiveRange:NULL][NSParagraphStyleAttributeName];
+
     boundingRect = [layoutManager usedRectForTextContainer:textContainer];
+    
+    // Work around line spacing of the last line not getting taken into account
```

Comment:
So is this a TextKit bug?  Were you able to find any other reports of it online?  Ideally we should file a Radar for this and reference the number here...almost certainly this code will break in a future version of iOS (if it is a workaround), and thus a future version of ourselves will be puzzling over the intent and context here.